src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -112,9 +111,7 @@ public SELF isBefore(String localDateTimeAsString) {
   public SELF isBeforeOrEqualTo(LocalDateTime other) {
     Objects.instance().assertNotNull(info, actual);
```

Comment:
this not necessary anymore as `comparables.assertIsBeforeOrEqualTo` does this check